documentation/src/main/markdown/packagemgrs/docker/scantarget.md

```diff
@@ -1,12 +1,10 @@
 # [solution_name]'s scan target
 
-When a Docker image is run; for example, using a `docker run` command, a container is created. That container has a file system; in other words, the container file system. The container file system at the instant the container is created; in other words, the initial container file system, can be determined in advance from the image without running the image. Because the target image is not yet trusted, Docker Inspector does not run the image; that is, it does not create a container from the image, but it does construct the initial container file system, which is the file system a container has at the instant it is created.
+When a Docker image is run; for example, using a `docker run` command, a container is created. That container has a file system; in other words, the container file system. When created, the initial container file system can be determined in advance from the image without running the actual image. Since the target image is not yet trusted, Docker Inspector does not run the image; that is, it does not create a container from the image, but it does construct the initial container file system.
```

Comment:
hmmm... I am not sure if I am seeing that in the same light, but how about this rewording:

_When a Docker image is run; for example, using a `docker run` command, a container is created that has an initial file system. This initial container file system can be determined in advance from the image without running the actual image. Since the target image is not yet trusted, Docker Inspector does not run the image; that is, it does not create a container from the image, but it does construct the initial container file system._

Reply:
That looks great to me.

Reply:
Done.